src/parser/core/modules/CooldownDowntime.tsx

```diff
@@ -210,6 +210,7 @@ export abstract class CooldownDowntime extends Analyser {
 				name: this.defenseChecklistName,
 				description: this.defenseChecklistDescription,
 				requirements: defensiveRequirements,
+				showProgress: false,
```

Comment:
This will turn off progress bars for all CDDT modules, which would be less than ideal, since most jobs use them for their offensive cooldowns, and it's preferred for them to still receive a progress bar.

Reply:
My mistake, this is actually fine.